src/queues/job-queue.js

```diff
@@ -151,7 +150,7 @@ const initQueue = () => {
   BetterSocialCronQueue.setCronCallback(generalDailyQueue, (job, done) =>
     BetterSocialCronQueue.process(job, done, {
       credderScoreQueue,
-      deleteActivityProcessQueue
+      removeActivityQueue
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The queue name `deleteActivityProcessQueue` hasn't been updated to reflect the new job name `removeActivityProcessJob`. This could lead to a runtime error if `deleteActivityProcessQueue` doesn't exist or isn't defined elsewhere in the code.
```diff
- removeActivityQueue
+ removeActivityProcessQueue
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->